sync2jira/downstream_issue.py

```diff
@@ -767,6 +767,16 @@ def _update_jira_issue(existing, issue, client):
         log.info("Looking for new transition(s)")
         _update_transition(client, existing, issue)
 
+    on_close_updates = None
```

Comment:
Could you add a comment on what we are syncing `# Only synchronize close labels for listings tat op-in?

Also, can we move lines 770-774 to inside the `_update_on_close` function? That way we can keep `_update_jira_issue` really clean?

Reply:
> Could you add a comment on what we are syncing `# Only synchronize close labels for listings that op-in?`
> 

Sure.

> Also, can we move lines 770-774 to inside the `_update_on_close` function? That way we can keep `_update_jira_issue` really clean?

Doing this would mean that the `issue_updates` list would be needed to be parsed 2 times: once in `if any(..)` and then once again in `_update_on_close()`, in order to extract the `on_close` dictionary from the list. Is doing this work twice okay?

Reply:
Or `_update_on_close()` could be called without a condition. But then it would not be visible from `_update_jira_issue` that this action as optional.

Reply:
Call `_update_on_close()` without a conditional, log something like `Attempting to sync close labels` and then inside `_update_on_close ()` log something like `Synced XYZ labels` if we were able to sync anything. 

This has been the workflow we've used for other fields.

Reply:
If you think it makes more sense, you can also check the conditional first. I will leave it up to you, but try to take as much code off our `_update_jira_issue`

Reply:
Updated to have an unconditional `_update_on_close()` call.

Reply:
Looks good, I'll test & merge if it passes later today.